dao/postgres/pg-utils.js

```diff
@@ -409,12 +418,18 @@ class PgUtils {
         });
     }
 
+    
     addFilterOptions(filter, fields, values, operators) {
         debug(`addFilterOptions()`);
         for (let fieldName in filter) {
             fields.push(fieldName);
-            values.push(`%${filter[fieldName]}%`);
-            operators.push('ILIKE');
+            if(filter[fieldName].indexOf('|') > 0){ 
+                values.push(`%(${filter[fieldName]})%`);
```

Comment:
This is probably not correct and possibly dangerous. Imaging you have a group called `dev` and one called `superdev` and the calling user has the group `dev`; the clause creates this `SIMILAR TO %(dev)%`, which would also match `superdev`. For this special case, i.e. matching a user's groups to the `api_group`, the `%` must be left out.

Reply:
Oh, and this also must be applied when the user only has one group! Otherwise you get the same effect again, via `ILIKE`. Or did I read this wrong?

Reply:
Single should work as well, it's just 'ILIKE' operator will take over for those cases

Reply:
Which is not correct, as it would match `dev` with `superdev`. It must be a specific thing here which should use the `=` operator, or `SIMILAR TO '(dev)'`, and not `ILIKE '%dev%'`

Reply:
oh yes

Reply:
fixed

Reply:
I don't think it's fixed, is it? In which way is it fixed?